feature-wallet-impl/src/main/java/jp/co/soramitsu/feature_wallet_impl/data/repository/WalletRepositoryImpl.kt

```diff
@@ -250,12 +242,8 @@ class WalletRepositoryImpl(
             }
     }
 
-    private suspend fun getAssetPrice(networkType: Node.NetworkType, request: AssetPriceRequest): SubscanResponse<AssetPriceStatistics> {
-        return try {
-            apiCall { subscanApi.getAssetPrice(networkType.subscanSubDomain(), request) }
-        } catch (_: Exception) {
-            SubscanResponse.createEmptyResponse()
-        }
+    private suspend fun getAssetPriceCoingecko(networkType: Node.NetworkType): Map<String, PriceInfo> {
+        return apiCall { coingeckoApi.getAssetPrice(networkType.toString(), "usd", true) }
```

Comment:
it is better to use named arguments (`includeRateChange=true`) when passing values which does not have representative name